leakpro/attacks/utils/gan_handler.py

```diff
@@ -0,0 +1,87 @@
+"""Module for handling GANs."""
+import os
+
+import torch
+from torch.nn import Module
+
+from leakpro.input_handler.abstract_input_handler import AbstractInputHandler
+from leakpro.utils.import_helper import Self
+from leakpro.utils.logger import logger
+
+from .generator_handler import GeneratorHandler
+
+
+class GANHandler(GeneratorHandler):
+    """Handler for training and managing GANs."""
+
+    def __init__(self: Self, handler: AbstractInputHandler, configs: dict) -> None:
```

Comment:
I think we should create a Minv handler as the train_gan is not required elsewhere.